zcash_keys/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this library adheres to Rust's notion of
 
 ## [Unreleased]
 
+## [0.4.1, 0.5.1, 0.6.1, 0.7.1, 0.8.1] - 2025-05-07
+
+### Added
+- `zcash_keys::Address::to_transparent_address`
```

Comment:
Ditto: 0.8.1 should be in the same group as 0.7.1 without this line.

(Also update the release dates when you make the post-release changelog cleanup.)